recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
 
 build:
   number: 0
```

Comment:
```suggestion
  number: 1
```

Reply:
Done. Thank you.

Reply:
It may not be needed here as it will produce the noarch package:
![image](https://user-images.githubusercontent.com/13209176/87062921-a8ab0400-c1db-11ea-9460-26d5e3c8dc03.png)

but to avoid potential confusion for conda to whether take an arch or noarch package with the same version and build number, it's better to increment it.